doc/man1/flux-resource.rst

```diff
@@ -61,15 +62,15 @@ COMMANDS
    by ``flux resource info`` is "all".
 
 **status**  [-n] [-o FORMAT] [-s STATE,...] [--skip-empty]
-   Show system view of resources.  The *-n,--no-header* suppresses header
-   from output, *-o,--format=FORMAT* customizes output formatting (see
-   below), and *-s,--states=STATE,...* limits output to specified resource
-   states, where valid states are "online", "offline", "avail", "exclude",
-   "draining", "drained", and "all". The special "drain" state is also
-   supported, and selects both draining and drained resources. Normally,
-   ``flux resource status`` skips lines with no resources, unless the
-   ``-s, --states`` option is used. Suppression of empty lines can always
-   be forced with the ``--skip-empty`` option.
+   Show system view of resources.  The *-n,--no-header* suppresses
+   header from output, *-o,--format=FORMAT* customizes output formatting
+   (see below), and *-s,--states=STATE,...* limits output to specified
+   resource states, where valid states are "online", "offline", "avail",
+   "exclude", "draining", "drained", "drained*", and "all". The special
+   "drain" state is also supported, and selects both draining and drained
+   resources. Normally, ``flux resource status`` skips lines with no
+   resources, unless the ``-s, --states`` option is used. Suppression of
+   empty lines can always be forced with the ``--skip-empty`` option.
```

Comment:
should we mention what `drain*` means?

Reply:
Hm, none of the states are described in this section. The description of `drained*` is in the section that describes drained nodes. We could reference that here?

Reply:
ahhh, i missed the description of `drained*` in the other section.  This is fine then.  Sorry for the noise :P

Reply:
NP, a pretty good suggestion IMO